cyber/tools/cyber_monitor/cyber_topology_message.cc

```diff
@@ -222,6 +222,9 @@ void CyberTopologyMessage::Render(const Screen* s, int key) {
     case SecondColumnType::MessageFrameRatio:
       s->AddStr(col1_width_ + SecondColumnOffset, 0, Screen::WHITE_BLACK,
                 "FrameRatio");
+    col2_width_ = 10;
+    s->AddStr(col1_width_ + col2_width_ + 2 * SecondColumnOffset, 0,
+	          Screen::WHITE_BLACK, "Bandwidth");                
```

Comment:
Plz fix indents.

Reply:
"col2_width_" is not necessary as one member. It's just one constant.